oem/ibm/libpldmresponder/file_io_type_dump.cpp

```diff
@@ -56,6 +53,8 @@ std::string DumpHandler::findDumpObjPath(uint32_t fileHandle)
         "org.freedesktop.DBus.ObjectManager";
     auto& bus = pldm::utils::DBusHandler::getBus();
 
+    resDumpDirPath = "/var/lib/pldm/resourcedump/" + std::to_string(fileHandle);
```

Comment:
this should be only set with dump type as PLDM_FILE_TYPE_RESOURCE_DUMP_PARMS as fileHandle has different meaning in other scenarios like offload

Reply:
done